packages/Takeover/src/components/FocusTrap.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
```

Comment:
Let's extract this component as well into the helpers seems a one that we want to reuse in more than one place.

Reply:
I will do this as part of work on Modal component. Does it work for you?

Reply:
ok make sense 👍